frontend/src/styles/Summary/Summary.scss

```diff
@@ -0,0 +1,29 @@
+@import '../root.scss';
+@import '../Button.scss';
+
+.summary-page{
+    display:flex;
+    flex-direction: column;
+    
+    align-content:center;
+}
+
+.league-dropdown-section{
+    padding-top:40px;
```

Comment:
Can we change this to vh and vw instead of px? Just to keep a consistent view for each screen size.

Reply:
okie dokes